demos/node-classification/node2vec/stellargraph-keras-node2vec-node-classification.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<p>I was under the impression that having <code>Node2Vec</code>  implemented with keras meant that we could do node classification via <code>Node2Vec</code>  in a single keras pipeline, instead of doing it separately after doing representation learning - is this not the case?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/536/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
Yeah! Actually, we can couple node2vec with a classification task within a single keras pipeline. Here, I just use the node classification to evaluate the quality of node embeddings learned by node2vec, with the expectation that keras node2vec achieves the similar classification performance to gensim node2vec. In the future pull request, I will try do node classification with node2vec in a single keras pipeline.